cron/sqs_status.conf

```diff
@@ -0,0 +1,7 @@
+[options]
+accesskey = <accesskey>
+secretkey = <secretkey>
+region = <region>
+taskexchange = sqs,comma,delimited,list
```

Comment:
We need the ability to specify an account owner, per region, per sqs queue. What we have right now, will not cover all of the queues that we care about (as we are restricting our "search" to only one region for one account).

Reply:
wouldn't it be the same logic/script, just a different -c conf file for each queue you are targeting?

Reply:
I'd argue that it'd be better to modify a config file to add a new queue, instead of having to append to crontab to run another python file.

Reply:
Then the same access key/secret key/region has to be valid for each queue?

Reply:
Yeah this is a concern, I was sort of modeling this off of how we currently do healthAndStatus, but what I noticed is that even healthAndStatus only seems to pull data and insert an event from a single host at a time, that data is not merged together in the result as I was trying to do here.

SQS queues are not host based, and I think that's where I'm finding some issue with how to extract this data and present it without it causing some logic problems.

I did want to avoid a separate config for each queue, but doing this will require a better schema structure, and if everything is in one event, then the event can possibly have an infinite amount of fields in it's mapping (around 3 or 4 for each queue).

Also to speak to @jeffbryner's point of accesskey/secretkey for all of them does cause problems, because we don't have the same key in multiple accounts. 

So with that said, I think moving back to the one config per queue addresses most of these problems, and if one queue fails, it doesn't fail for all of them in this manner.

I can add ownership information for each queue as well to address that need.

I do feel there is a more elegant way to do this, but not with crons, I'm thinking more of a microservice implementation that would run on supervisord to kick off these checks and feed into ES using celery maybe. Could possibly move to that kind of structure for all of our crons and get out of using cron and simply add the queue name, keys, region, and owner to a list in a celery config, or use existing esworker configs to grab the required data and populate the values needed for the connection. 
So instead of having a cron directory for these items, maybe we just have a plugin that cycles through the mq configs and pulls this data using celery workers?

Reply:
Would it be better to represent the config file in a json format? So that you can do something like:

```
queues = [
    {
        "queue_name" : "ExampleQueue",
        "access_key": "12345",
        "secret_key": "abcdefg",
        "region": "us-west1",
    },
    {
        "queue_name" : "AnotherExampleQueue",
        "access_key": "678910",
        "secret_key": "hijklmnop",
        "region": "us-west2",
    }
]
```

Reply:
Going to try the json config and see if I can make that work.